### PR TITLE
PMM-9617 Add helper to generate dashboard url with desired params

### PR DESCRIPTION
**Pull request by ElenaFedichkina**
added new method to create url with params

https://pmm.cd.percona.com/job/pmm2-ui-test-nightly/1068/
image

### CODE REVIEW

File: tests/custom_steps.js [hunk 1]
```diff
@@ -76,4 +77,55 @@ module.exports = () => actor({
...
+          body['var-environment'] = value;
+          break;
+        case 'node_name':
+          body['var-node_name'] = value;
+          break;
+        case 'service_name':
+          body['var-service_name'] = value;
+          break;
+        case 'columns':
+          value ? body.columns = value : body.columns = '%5B%22load%22,%22num_queries%22,%22query_time%22%5D';
```
Comment (yurkovychv) [inline tests/custom_steps.js:108]: why do we need this special list of columns? isn't it a default one?

File: tests/custom_steps.js [hunk 2]
```diff
@@ -76,4 +77,55 @@ module.exports = () => actor({
...
+   *
+   * @param url start
+   * @param parameters object
+   * @returns {Promise<void>}
+   *
+   * @example
+   * getDashboardUrlWithParams('http://example.com', { environment: 'ps-dev', from: 'now-1' });
+   */
+  getDashboardUrlWithParams(url, parameters) {
+    const body = {};
```
Comment (yurkovychv) [inline tests/custom_steps.js:92]: please rename it to `queryParams` and just use `{ queryParams }` in `buildUrl` function

File: tests/custom_steps.js [hunk 3]
```diff
@@ -76,4 +77,55 @@ module.exports = () => actor({
...
+   * Create URL method
+   *
+   * @param url start
+   * @param parameters object
+   * @returns {Promise<void>}
+   *
+   * @example
+   * getDashboardUrlWithParams('http://example.com', { environment: 'ps-dev', from: 'now-1' });
+   */
+  getDashboardUrlWithParams(url, parameters) {
```
Comment (yurkovychv) [inline tests/custom_steps.js:91]: please rename it to `buildUrlWithParams`

File: tests/dashboards/verifyMysqlDashboards_test.js [hunk 4]
```diff
@@ -36,9 +36,10 @@ Scenario(
     I, adminPage, dashboardPage,
   }) => {
     for (const serviceName of serviceList) {
-      I.amOnPage(dashboardPage.mysqlInstanceSummaryDashboard.url);
+      const url = I.getDashboardUrlWithParams(dashboardPage.mysqlInstanceSummaryDashboard.url, null, serviceName, 'now-5m', 'now');
```
Comment (yurkovychv) [inline tests/dashboards/verifyMysqlDashboards_test.js:39]: As I see this line will fail

File: tests/custom_steps.js (hunk 1 above)
Comment (ElenaFedichkina) [reply tests/custom_steps.js:108]: yes, body.columns = '%5B%22load%22,%22num_queries%22,%22query_time%22%5D' - it is default value for columns.

File: tests/custom_steps.js (hunk 3 above)
Comment (ElenaFedichkina) [reply tests/custom_steps.js:91]: ok

File: tests/custom_steps.js (hunk 2 above)
Comment (ElenaFedichkina) [reply tests/custom_steps.js:92]: edited

File: tests/dashboards/verifyMysqlDashboards_test.js (hunk 4 above)
Comment (ElenaFedichkina) [reply tests/dashboards/verifyMysqlDashboards_test.js:39]: fixed

File: tests/custom_steps.js (hunk 1 above)
Comment (yurkovychv) [reply tests/custom_steps.js:108]: we don't need to hardcode it then. It is appended automatically by the app itself

Comment (ElenaFedichkina) [reply tests/custom_steps.js]: ok. removed it

File: tests/QAN/details_test.js [hunk 5]
```diff
@@ -4,10 +4,7 @@ const { adminPage } = inject();
 
 Before(async ({ I, qanPage, qanOverview }) => {
   await I.Authorize();
-  I.amOnPage(qanPage.url);
-
-  qanOverview.waitForOverviewLoaded();
-  await adminPage.applyTimeRange('Last 1 hour');
+  I.amOnPage(I.buildUrlWithParams(qanPage.clearUrl, { from: 'now-1h' }));
```
Comment (yurkovychv) [inline tests/QAN/details_test.js:7]: `now-5m` here and in other similar places

**Review by yurkovychv — APPROVED** [review]

**Pull request merged**